core/trino-main/src/main/java/io/trino/operator/ExchangeClient.java

```diff
@@ -274,13 +274,13 @@ private synchronized void scheduleRequestIfNecessary()
         return buffer.isBlocked();
     }
 
-    private boolean addPages(HttpPageBufferClient client, List<SerializedPage> pages)
```

Comment:
No need to change the variable name. It is understood that they are serialized pages since they are represented as Slice

Reply:
Actually I would vote for renaming (but I do not feel strongly). With more percise variable name it is nicer to read to me and does not require to look at variable type.

Reply:
I find reading code with longer and verbose variable names harder (especially when the type name is repeated in the variable name — Hungarian notation?), so I try to avoid them unless it’s not immediately obvious from context what they are, or if the meaning is ambiguous due to other conflicting or similar names

Reply:
Oh, I see your point. Since the `Slice` type represents a "blob" it already suggests that this is rather "something" that is serialized. And to clarify what is "something" naming variables as "page / pages" should be sufficient. Let me change the naming back then.